packages/apps/composer-app/src/playwright/playwright-smoke.config.cts

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { nxE2EPreset } from '@nx/playwright/preset';
+import { defineConfig } from '@playwright/test';
+
+import { e2ePreset } from '@dxos/test-utils/playwright';
+
+export default defineConfig({
+  ...nxE2EPreset(__filename, { testDir: __dirname }),
+  ...e2ePreset(__dirname),
+  testMatch: '**/*.smoke.ts',
+  timeout: 5 * 60_000, // 5 minutes
```

Comment:
I wouldn't make this longer than it needs to be because if they do get stuck the process will be running for 5 minutes for each test and they aren't the cheapest. Based on the screenshot you sent me I think 90s should be fine